apps/src/storage/firebaseStorage.js

```diff
@@ -451,18 +451,22 @@ FirebaseStorage.populateTable = function (jsonData, overwrite, onSuccess, onErro
   if (!jsonData || !jsonData.length) {
     return;
   }
-  getExistingTables(overwrite).then(existingTables => {
-    const promises = [];
-    const newTables = JSON.parse(jsonData);
-    for (const tableName in newTables) {
-      if (overwrite || (existingTables[tableName] === undefined)) {
-        const newRecords = newTables[tableName];
-        const recordsData = getRecordsData(newRecords);
-        promises.push(overwriteTableData(tableName, recordsData));
+  // Ensure rate limit counters have been initialized, so that updates to the
+  // counters/tables node will pass type definition checks in the security rules.
+  incrementRateLimitCounters()
+    .then(() => getExistingTables(overwrite))
+    .then(existingTables => {
+      const promises = [];
+      const newTables = JSON.parse(jsonData);
```

Comment:
for example:
<img width="431" alt="screen shot 2017-02-07 at 7 00 34 pm" src="https://cloud.githubusercontent.com/assets/8001765/22721949/966cf4f6-ed68-11e6-9576-7cd3fffa02a1.png">